test/wormhole/wormhole.ts

```diff
@@ -112,6 +122,12 @@ export async function configureWormhole({
   await executeSpell(sdk, configureSpell)
 
   for (const [domainName, domainCfg] of Object.entries(domainsCfg)) {
```

Comment:
We could maybe drop the loop and the usage of a `domainsCfg` dictionary with domain keys in favour of directly referring to the domain variables (line, etc). Looping over multiple domains does not seem very useful at the moment given that tests for different domains are ran independently.

But if we decide to keep the loop pattern, then we probably need to also include l2WormholeBridge, l2Dai and l1Escrow to the domainsCfg dictionary.